lib/browser-runtime.ts

```diff
@@ -775,6 +780,31 @@ function beforeHandler(context: CompositionContext) {
   taskSpecEnvelopes(context);
 }
 
+function beforeAllHandler(this: Mocha.Context, context: CompositionContext) {
```

Comment:
This method's content can be moved into `beforeHandler(..)`.

Reply:
Refactored it